### PR TITLE
Revert "Rename form 28-8832 to 27-8832"

### PR DESCRIPTION
**Pull request by zack**
We went about this all the wrong way.

This reverts commits:
- 8fe4f0622ac13ab5105bec76f050779b315363b1
- 81fe1863f7280db3640138888900d1d59cfa4ecc

### CODE REVIEW

File: dist/version.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "version": "21.2.0"
+  "version": "21.1.4"
```
Comment (zack) [inline dist/version.json:2]: Not sure if we should revert this or bump it again?

Comment (LindseySaari) [reply dist/version.json:2]: I think we would have to "fail foward"

Comment (zack) [reply dist/version.json:2]: 👍

**Comment by zack** [comment]
Thanks @LindseySaari. Sorry for all the noise today and yesterday. :(

File: dist/version.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-  "version": "21.2.0"
+  "version": "21.3.0"
```
Comment (LindseySaari) [inline dist/version.json:2]: Does the version and package.json need to be the same?

**Review by tblackwe — APPROVED** [review]

**Comment by zack** [comment]
Hi @LindseySaari, is everything in order? We'd like to merge this to unblock another team. Thanks!

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**